db/db_test2.cc

```diff
@@ -2365,10 +2368,49 @@ TEST_F(DBTest2, CompressionManagerCustomCompression) {
     ASSERT_EQ(Get("d").size(), kValueSize);
     ASSERT_EQ(Get("e").size(), kValueSize);
 
-    // TODO: mix of compatibility names in same DB
+    // Add a file using mgr_bar
```

Comment:
Related to your previous PR -  "...// Verify it was compressed with snappy" in line 2326 above should be with LZ4